.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,32 @@
+---
+name: 🙋 New Feature
+about: Do you want to add something to parcel?
+---
+
+<!---
+Thanks for filing a pull request 😄 ! Before you submit, please read the following:
+
+Search open/closed issues before submitting since someone might have pushed the same thing before!
+-->
+
+# ✨ New Feature
+
+<!---
+Provide a general summary of the feature here
+Does this address an existing feature request?
+
+Tell us how the feature should work
+-->
+
+## 🔦 Context
+
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## 💻 Examples
+
+<!-- Examples help us understand the requested feature better -->
+
```

Comment:
Probably also a good idea to add a checklist to the end of the PR to make sure they pass all the necessary requirements for a PR, something like this:

- [ ] Added/updated unit tests for this change
- [ ] Filled out test instructions
- [ ] Included links to related issues/PRs